sorc/orog_mask_tools.fd/orog.fd/CMakeLists.txt

```diff
@@ -25,10 +25,8 @@ target_include_directories(orog_lib INTERFACE ${mod_dir})
 target_link_libraries(
   orog_lib
   PUBLIC
-  bacio::bacio_4
   w3emc::w3emc_d
   ip::ip_d
-  sp::sp_d
```

Comment:
@AlexanderRichert-NOAA - loading ip also automatically loads sp as well. So, I did not think was needed. Should I add it back?

Reply:
That makes sense, though I haven't tested it. As far as the root CMakeLists.txt, I would imagine that it would fail without sp available, though I haven't tested that either.

Reply:
I will add back the SP dependency. I can update the build once the merged IP library is installed.